src/felles-komponenter/EkspanderbartTekstomrade.tsx

```diff
@@ -0,0 +1,45 @@
+import { NedChevron, OppChevron } from 'nav-frontend-chevron';
+import React, { useState } from 'react';
+
+import styles from './EkspanderbartTekstomrade.module.less';
+
+const MoreOrLess = ({ text, maxCharacters }: { text: string; maxCharacters: number }) => {
+    const [hasLongText, setHasLongText] = useState(true);
+    const toggleMoreOrLess = () => {
+        setHasLongText(!hasLongText);
+    };
+    if (text.length > maxCharacters) {
+        return (
+            <p>
+                {hasLongText ? text.slice(0, maxCharacters) + ' ... ' : text + ' '}
```

Comment:
Kan det stemme at det siste mellomrommet der er unødvendig?
```suggestion
                {hasLongText ? text.slice(0, maxCharacters) + ' ... ' : text}
```

Reply:
Brukes for å skille mellom innholdsteksten og "Vis mindre", ut fra tanken om at linja som legger på label ikke trenger å forholde seg til spacing. Kan fikse.